integration_tests/e2e/createContactsCheckAnswersChangeDob.cy.ts

```diff
@@ -0,0 +1,218 @@
+import Page from '../pages/page'
+import EnterNamePage from '../pages/enterNamePage'
+import CreatedContactPage from '../pages/createdContactPage'
+import EnterContactDateOfBirthPage from '../pages/enterContactDateOfBirthPage'
+import CreateContactCheckYourAnswersPage from '../pages/createContactCheckYourAnswersPage'
+import EnterContactEstimatedDateOfBirthPage from '../pages/enterContactEstimatedDateOfBirthPage'
+import SelectRelationshipPage from '../pages/selectRelationshipPage'
+import TestData from '../../server/routes/testutils/testData'
+import SelectEmergencyContactPage from '../pages/selectEmergencyContactPage'
+import SelectNextOfKinPage from '../pages/selectNextOfKinPage'
+
+context('Create contact and update from check answers where we are changing the DOB', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn', { roles: ['PRISON'] })
+    cy.task('stubComponentsMeta')
+    cy.task('stubTitlesReferenceData')
+    cy.task('stubRelationshipReferenceData')
+    cy.task('stubPrisonerById', TestData.prisoner())
+    cy.task('stubCreateContact', { id: 132456 })
+
+    cy.signIn()
+    cy.visit('/prisoner/A1234BC/contacts/create/start')
+    Page.verifyOnPage(EnterNamePage) //
+      .enterLastName('Last')
+      .enterFirstName('First')
+      .continueTo(SelectRelationshipPage, 'Last, First')
+      .selectRelationship('MOT')
+      .continueTo(SelectEmergencyContactPage, 'Last, First')
+      .selectIsEmergencyContact('NO')
+      .continueTo(SelectNextOfKinPage, 'Last, First')
+      .selectIsNextOfKin('NO')
+      .continueTo(EnterContactDateOfBirthPage, 'Last, First')
```

Comment:
Example chaining using `continueTo` when you want to be more concise like in a pre-step here.